adapter_pipelines/ss2_single_sample/adapter.wdl

```diff
@@ -0,0 +1,212 @@
+import "ss2_single_sample.wdl" as ss2
+import "submit.wdl" as submit_wdl
+
+
+task GetInputs {
+  String bundle_uuid
+  String bundle_version
+  String dss_url
+  Int retry_seconds
+  Int timeout_seconds
+
+  command <<<
+    python <<CODE
+    from pipeline_tools import dcp_utils
+
+    # Get bundle manifest
+    uuid = "${bundle_uuid}"
+    version = "${bundle_version}"
+    dss_url = "${dss_url}"
+    retry_seconds = ${retry_seconds}
+    timeout_seconds = ${timeout_seconds}
+    print("Getting bundle manifest for id {0}, version {1}".format(uuid, version))
+    manifest_files = dcp_utils.get_manifest_files(uuid, version, dss_url, timeout_seconds, retry_seconds)
+
+    print("Downloading assay.json")
+    assay_json_uuid = manifest_files["name_to_meta"]["assay.json"]["uuid"]
+    assay_json = dcp_utils.get_file_by_uuid(assay_json_uuid, dss_url)
+
+    sample_id = assay_json["has_input"]
+    fastq_1_name = assay_json["content"]["seq"]["lanes"][0]["r1"]
+    fastq_2_name = assay_json["content"]["seq"]["lanes"][0]["r2"]
+    fastq_1_url = manifest_files["name_to_meta"][fastq_1_name]["url"]
+    fastq_2_url = manifest_files["name_to_meta"][fastq_2_name]["url"]
+
+    print("Creating input map")
+    with open("inputs.tsv", "w") as f:
+        f.write("fastq_1\tfastq_2\tsample_id\n")
+        f.write("{0}\t{1}\t{2}\n".format(fastq_1_url, fastq_2_url, sample_id))
+    print("Wrote input map")
+    CODE
+  >>>
+  runtime {
+    docker: "humancellatlas/pipeline-tools:0.1.5"
+  }
+  output {
+    Object inputs = read_object("inputs.tsv")
+  }
+}
+
+workflow AdapterSmartSeq2SingleCell{
+  # variable parameters from blue
```

Comment:
I think this could be more descriptive here, like "variable parameters from the notification sent by the data storage service"